### DISCUSSION TITLE
Not serving static SVG files

### DISCUSSION THREAD

**Discussion by huntc** [parcel-bundler/parcel#6622] [Q&A]
I've placed an svg file in my `static/images` folder but it isn't able to be accessed via an `img` tag. Does Parcel require some special configuration to serve SVG files? Thanks for any help.

**Comment by DeMoorJasper** [comment]
You can probably use the proxy to serve a static files folder https://v2.parceljs.org/features/api-proxy/

**Reply by huntc** [reply]
Thanks for replying. It is interesting to me that I'd need to specify the additional configuration. Also, for the record, png files from my `static/images` folder aren't being served either.

So I get that I could proxy to another server, however it seems a bit more complex to have a server for just static content.

Any further thinking on why my `static/images` files are not being served up by the built-in server? Thanks.

**Reply by huntc** [reply]
Also, the image is coming back as html...

![image](https://user-images.githubusercontent.com/694893/126463973-f3f724b0-a25c-4010-adb2-adea66776da2.png)

**Reply by huntc** [reply]
...and the images don't exist in the `dist` folder... (but the `static/css` files do, along with the `favicon.ico`).

**Reply by mischnic** [reply]
How are you using these static files? Inserting them via JS? If they are specified in an img tag like here: https://v2.parceljs.org/languages/html/ they would be automatically detected and copied

**Reply by huntc** [reply]
Ah, of course. The image files are specified dynamically. I’m using wasm and Yew to generate the dom. How is this situation dealt with? I imagine it’d be the same with React… Thanks.

**Reply by mischnic** [reply]
From JS, you import static assets via `import x from "./image.png";` or `import y from "url:./foo.mp4";`

**Reply by huntc** [reply]
Makes sense now. Thanks.

**Reply by huntc** [reply]
I don’t know of an equivalent import expression for Yew/parcel integration. I’m suspecting that I’ll have to associate the image via css as Parcel understands css files. Thanks for all of the follow-ups!

**Reply by mischnic** [reply]
The "proper" solution would probably be writing a new Parcel plugin.
